### PR TITLE
Update Working-Draft.md

### PR DESCRIPTION
**Pull request by ignpelloz**
Updated Working-Draft.md to conform to the schema added in #24.

### CODE REVIEW

File: versions/Working-Draft.md [hunk 1]
```diff
@@ -30,15 +31,23 @@ info:
...
-A **SLA4API** document is built as an extension to a given [OAS](https://github.com/OAI/OpenAPI-Specification) document. This extension will add an SLA definition to all or part of the services exposed in the API by means of different plans that include limitations over the usage of the API. Specifically, a full SLA definition is a `YAML` (or `JSON`) document composed of the structure of an [SLA Object](#sla-object).
+
+A **SLA4OAI** document is built as an extension to a given [OAS](https://github.com/OAI/OpenAPI-Specification) document. This extension will add an SLA definition to all or part of the services exposed in the API by means of different plans that include limitations over the usage of the API. Specifically, a full SLA definition is a `YAML` (or `JSON`) document composed of the structure of an [SLA Object](#sla-object).
 
 
 **Simple Example:**
 
 ```yaml
-sla: 1.0.0
+sla4oai: 1.0.0
```
Comment (pjmolina) [inline versions/Working-Draft.md]: Use OAS instead of OAI for consistency.  `sla4oas`

File: versions/Working-Draft.md [hunk 2]
```diff
@@ -30,15 +31,23 @@ info:
...
+2. The customer chooses the levels (plans) they need, then both provider and customer agree on the **SLA Agreement** document. See an example at [pro-petstore-sla.yml](./samples/petstore/pro-petstore-sla.yml).
+
 
 ##  Specification
-A **SLA4API** document is built as an extension to a given [OAS](https://github.com/OAI/OpenAPI-Specification) document. This extension will add an SLA definition to all or part of the services exposed in the API by means of different plans that include limitations over the usage of the API. Specifically, a full SLA definition is a `YAML` (or `JSON`) document composed of the structure of an [SLA Object](#sla-object).
+
+A **SLA4OAI** document is built as an extension to a given [OAS](https://github.com/OAI/OpenAPI-Specification) document. This extension will add an SLA definition to all or part of the services exposed in the API by means of different plans that include limitations over the usage of the API. Specifically, a full SLA definition is a `YAML` (or `JSON`) document composed of the structure of an [SLA Object](#sla-object).
```
Comment (pjmolina) [inline versions/Working-Draft.md]: SLA4OAI  rename to SLA4OAS- Fix in all places for consistency. 
Pablo to rename the repo also with forwarding.

File: versions/Working-Draft.md [hunk 3]
```diff
@@ -16,7 +16,8 @@ This specification is an on-going work developed by the [SLA4OAI Technical Commi
 
 |Version  | Date         | Notes            |
 |:------- |:------------ |:---------------- |
-| 1.0.0     | 2020-11-16 | Initial draft.   |
+| 1.0.0   | 2020-11-16   | Initial draft.   |
+| 1.0.1   | 2022-06-08   | Updates.         |
```
Comment (pjmolina) [inline versions/Working-Draft.md]: Explain the list of updates, please.

**Review by pjmolina — CHANGES_REQUESTED** [review]
Asked for changes to use OAS instead of OAI for consistency with OpenAPI Spec

**Comment by ignpelloz** [comment]
Hi @pjmolina I have updated Working-Draft.md accordingly. I will open another PR to update the schema: sla4oai rename to sla4oas.

**Review by pjmolina — APPROVED** [review]
looks good. Tim and Pedro J.

**Pull request merged**